lib/pre-build.js

```diff
@@ -212,13 +212,14 @@ function download(target, cb) {
   let client = null
   let hasCalledBack = false
   let options = {}
+  const proxyHost = process.env.NR_NATIVE_METRICS_PROXY_HOST
 
-  if (process.env.NR_NATIVE_METRICS_PROXY_HOST) {
-    const proxy = process.env.NR_NATIVE_METRICS_PROXY_HOST
-    options.host = new URL(DOWNLOAD_HOST).host
+  if (proxyHost) {
+    const parsedUrl = new URL(DOWNLOAD_HOST)
```

Comment:
I uncovered this bug while writing tests.  The `NR_NATIVE_METRICS_DOWNLOAD_HOST` can only be a hostname and not specify a port.  This prob only affects us but makes it impossible to specify an alternate host + port.